gui.js

```diff
@@ -3703,6 +3714,7 @@ IDE_Morph.prototype.rawOpenProjectString = function (str) {
         );
     }
     this.stopFastTracking();
+    this.updateDocumentTitleText();
```

Comment:
I *did!* Look at the lines of code before `);` - it's not *actually* inside `setExtent`, that's just where GitHub (git?) put the diff. Inconveniently.

![Image](https://cloud.githubusercontent.com/assets/9948030/16765384/1c0edebc-480a-11e6-847e-eb77d8f7b980.png)